server/policies/team.ts

```diff
@@ -32,3 +32,20 @@ allow(User, ["delete", "audit"], Team, (actor, team) =>
     isTeamAdmin(actor, team)
   )
 );
+
+allow(User, ["createDocument", "readDocument"], Team, (actor, team) =>
+  and(
+    //
+    !actor.isGuest,
+    !actor.isViewer,
+    isTeamModel(actor, team)
+  )
```

Comment:
We lost the `isTeamMutable` check here when you moved it from `policies/document`

In general I'm worried these could be confusing, thoughts on renaming to `createTemplate` etc?

Reply:
Yeah, that was intentional because `isTeamMutable` defaults to `true`. Is the `isTeamMutable` implementation a bug?

re: renaming, yes `template` is much better. I'll update the PR.

Reply:
Not a bug, it is used in private fork to control access

Reply:
Ah right!!